arenadata/Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN yum makecache && \
     yum -y install openssl-devel pam-devel readline-devel snappy-devel libuv-devel && \
     yum -y install apache-ivy libicu perl-ExtUtils-Embed perl-Env perl-JSON && \
     yum -y install perl-IPC-Run perl-Test-Base libxslt-devel && \
+    yum -y install libzstd-static-1.5.5-1.el7 && \
```

Comment:
may be simplify to
```sh
yum -y install libzstd-static
```
?

Reply:
We use experimental zstd function which api can change in future, so we need to fix the version that we use